OP2Helper.cpp

```diff
@@ -18,6 +19,13 @@ const ResourceSet MultiResourceSet = {
 	}
 };
 
+// Perform recordFunction discretely along a line or L shape
+void ExecuteAcrossLine(LOCATION loc1, LOCATION loc2, const std::function <void(LOCATION&)>& recordFunction);

```

Comment:
passing LOCATION& here would be more appropriate as const LOCATION&, but underlying Outpost2DLL does not support.